### PR TITLE
Add Kingdom Hearts 3D: Dream Drop Distance

### PR DESCRIPTION
**Pull request by OrangSquid**
Adds Kingdom Hearts 3D: Dream Drop Distance.

*Automatically generated by PR builder*

### CODE REVIEW

**Comment by chris062689** [comment]
Please add a testcase to this entry.

File: kingdom-hearts-3d-dream-drop-distance/game.dat [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+region = "eur"
+release_date = "2012-07-20"
+
+[[ testcases ]]
+title = "0004000000095500"
+compatability = "1"
+date = "2017-07-06"
+version = "HEAD-c017065"
+author = "OrangSquid"
+cpu = "Intel Core i7-6700HQ @ 2.60 GHz
```
Comment (TheKoopaKingdom) [inline kingdom-hearts-3d-dream-drop-distance/game.dat]: You're not ending the string here with a quote.

File: kingdom-hearts-3d-dream-drop-distance/game.dat [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+release_date = "2012-07-31"
+
+[[ releases ]]
+title = "0004000000095500"
+region = "eur"
+release_date = "2012-07-20"
+
+[[ testcases ]]
+title = "0004000000095500"
+compatability = "1"
```
Comment (TheKoopaKingdom) [inline kingdom-hearts-3d-dream-drop-distance/game.dat]: This should be `compatibility`, not `compatability`.

**Comment by TheKoopaKingdom** [comment]
LGTM

**Pull request merged**